zcash_client_backend/src/sync.rs

```diff
@@ -138,7 +123,7 @@ where
 
     // 7) Loop over the remaining suggested scan ranges, retrieving the requested data
     //    and calling `scan_cached_blocks` on each range.
-    let scan_ranges = db_data.suggest_scan_ranges()?;
+    let scan_ranges = wallet_data.suggest_scan_ranges()?;
```

Comment:
Same thing here. The unnecessary rename from `db_data` to `wallet_data` has caused this entire file to be conflicted, but because it was done in the same commit as the `BlocksCache` change, I can't just blow the commit away and recreate the rename.